packages/boa/tests/fixtures/es-module-loaders/common.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-process-exit */
+
+'use strict';
+
+const path = require('path');
+const { test } = require('tap');
+const { spawnSync } = require('child_process');
+
+const FLAG = '--experimental-loader';
+const PATH_ESM_LOADER = path.join(__dirname, '../../../esm/loader.mjs');
+const PATH_EXPORT_DEFAULT = path.join(__dirname, 'test-esm-loader-export-default.mjs');
+
+// See https://github.com/nodejs/node/pull/29796
+if (process.version < 'v12.11.1') {
+  console.log(`1..0 # Skipped: Current nodejs version: ${
+    process.version} does not support \`--experimental-loader\``);
+  process.exit(0);
+}
+
+function checkExec(t, appPath) {
+  const options = { encoding: 'utf8', stdio: 'inherit' };
+  const args = [FLAG, PATH_ESM_LOADER];
+  // See https://github.com/nodejs/node/pull/31974
+  if (process.version < 'v14.0.0') {
+    args.push('--experimental-modules');
+  }
+  const result = spawnSync(process.execPath, args.concat(appPath), options);
```

Comment:
This might be unneeded, it's recommended that adding a new script at https://github.com/alibaba/pipcook/blob/master/packages/boa/package.json#L9.